packages/components/addon/components/hds/table/th-button-tooltip.hbs

```diff
@@ -0,0 +1,14 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<button
+  type="button"
+  class="hds-table__th-button hds-table__th-button--tooltip"
+  {{hds-tooltip @text}}
+  aria-label="more information"
```

Comment:
@MelSumner is this OK? do you think we need to do something different or something else?